src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -1730,6 +1730,50 @@ def test_ps_s3_creation_triggers_on_master_ssl():
         del os.environ['RABBITMQ_CONFIG_FILE']
 
 
+def _get_body(response):
+    body = response['Body']
+    got = body.read()
+    if type(got) is bytes:
+        got = got.decode()
+    return got
+
+
+@attr('sample_test')
+def test_http_post_object_upload():
+    """ test that uploads object using HTTP POST """
+
+    import boto3
+    from collections import OrderedDict
+    import requests
+
+    #conn = connection()
+
+    conn1 = boto3.client(service_name='s3',aws_access_key_id=get_access_key(),aws_secret_access_key=get_secret_key())
+
+    # create bucket
+    bucket_name = gen_bucket_name()
+    #bucket = conn.create_bucket(bucket_name)
+
+    key_name = 'foo.txt'
+
+    resp = conn1.generate_presigned_post(Bucket=bucket_name, Key=key_name,)
```

Comment:
you need to create the connection with RGW not the AWS server

Reply:
see URL in: https://paste.sh/7R2S4FyM#DlbZmQuLlRyw2xdLnr4V2rNa
`https://ruuixb-1.s3.amazonaws.com/`